packages/discord.js/src/structures/BaseGuildTextChannel.js

```diff
@@ -22,7 +22,7 @@ class BaseGuildTextChannel extends GuildChannel {
 
     /**
      * A manager of the threads belonging to this channel
-     * @type {ThreadManager}
+     * @type {GuildTextThreadManager}
      */
     this.threads = new ThreadManager(this);
```

Comment:
Continuation from before
```suggestion
    this.threads = new GuildTextThreadManager(this);
```